streams/src/test/java/org/apache/kafka/streams/kstream/internals/SuppressScenarioTest.java

```diff
@@ -459,6 +460,89 @@ public void shouldSupportFinalResultsForTimeWindowsWithLargeJump() {
         }
     }
 
+    @Test
+    public void shouldSupportFinalResultsForSlidingWindows() {
```

Comment:
Aside from join, forgetting to test new operators in front of Suppress has also been an issue. It's great to see this test here!